strongbox-commons/pom.xml

```diff
@@ -113,11 +113,12 @@
 
         <dependency>
             <groupId>org.carlspring.commons</groupId>
-            <artifactId>commons-io</artifactId>
+            <artifactId>commons-http</artifactId>
         </dependency>
+
         <dependency>
             <groupId>org.carlspring.commons</groupId>
-            <artifactId>commons-http</artifactId>
+            <artifactId>commons-io</artifactId>
         </dependency>
```

Comment:
Is there any specific reasoning for this change?

Reply:
@steve-todorov : Ignore this. It's coming from @ptirador 's merged request on multiple byte ranges ( #1474 ) and is noise caused the merge.

Reply:
Ahh.. I see - i'll mark it as resolved then since it's not a real issue. :)